opentelekomcloud/services/vpn/resource_opentelekomcloud_vpnaas_ipsec_policy_v2.go

```diff
@@ -66,9 +78,11 @@ func ResourceVpnIPSecPolicyV2() *schema.Resource {
 			"transform_protocol": {
 				Type:     schema.TypeString,
 				Optional: true,
-				Computed: true,
+				Default:  "esp",
+				ValidateFunc: validation.StringInSlice([]string{
+					"esp", "ah", "ah-esp",
+				}, false),
```

Comment:
Why is it not computed any more?

Reply:
Cannot be used when Default is specified